tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -65,213 +65,212 @@ def header_colors():
-                                        max=3,
-                                        handleLabel={'showCurrentValue': True, 'label': ' '},
-                                        step=0.01,
-                                        marks={str(num): str(num) for num in range(4)}
-                                    ),
-                                    dcc.Input(
-                                        className='vp-test-util-div',
-                                        id='vp-upper-bound-val',
-                                        value=1,
-                                        type='number',
-                                    )
-                                ],
-                            ),
-                            html.Div(
-                                className='vp-vertical-style',
-                                title='Move horizontal dashed line',
-                                children=[
-                                    html.Div(
-                                        "Threshold",
-                                        className='vp-text',
-                                    ),
-                                    daq.Slider(
-                                        className='vp-input',
-                                        id='vp-genomic-line',
-                                        value=4,
-                                        max=10,
-                                        min=0,
-                                        handleLabel={'showCurrentValue': True, 'label': ' '},
-                                        step=0.01,
-                                        marks={str(num): str(num) for num in range(0, 11, 2)}
-
-                                    ),
-                                    dcc.Input(
-                                        className='vp-test-util-div',
-                                        id='vp-genomic-line-val',
-                                        value=4,
-                                        type='number',
-                                    )
-                                ],
-                            ),
-                        ],
-                    ),
-                    html.Div(
-                        id='vp-indicators-div',
-                        children=[
+    return html.Div(id='vp-page-content', children=[
+        html.Div(
+            id='vp-graph-div',
+            children=dcc.Graph(
+                id='vp-graph'
+            ),
+        ),
+        html.Div(id='vp-control-tabs', children=[
+            dcc.Tabs(id='vp-tabs', value='what-is', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='vp-tab', children=[
+                        html.H4('What is Volcano Plot?'),
+                        html.P(
+                            'You can use volcano plot to interactively '
```

Comment:
```suggestion
                            'You can use Volcano Plot to interactively '
```
(or, alternatively, "a volcano plot")
I would also drop "you can"